src/Stryker.Core/Stryker.Core.UnitTest/Mutants/TestResources/Mutator_FromActualProject_OUT.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Test 
```

Comment:
From what actual project is this code and do we have correct attribution /licensing to use it?

Reply:
this is some snippets from NFluent. You want me to state it explicitly?

Reply:
I thought it might be NFluent but I wasn't sure. If NFluent doesn't require attribution then this is fine. Since it explicitly states it is code from an actual project I just thought I would double check with you that we can use it.

Reply:
I kinda of seen this as fair use, as it was only a couple of methods. My intent was to make room to have tests that would work on 'known to have failed' code constructs.